<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepROV</class>
 <widget class="QDialog" name="DeepROV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="LoadImg_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ImgName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>370</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadWeight_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Pretrained Weight</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WeightName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Detection_pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Detection!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StatusBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="jsonName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Loadjson_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load json</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
